ui/src/components/Fields/time-ago-field.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react'
+import cx from 'classnames'
+import { RepoSyncDataType } from 'src/@types'
+import { getTimeAgoFromNow } from 'src/utils'
+
+export type TimeAgoFieldProps = {
+  date: string,
+  syncData?: RepoSyncDataType,
+  extraStyles?: string
+}
+
+export const TimeAgoField: React.FC<TimeAgoFieldProps> = ({ date, syncData, extraStyles }) => {
+  const [dateRun, setDateRun] = useState<string>('')
+
+  const recalculateTimeAgo = () => {
+    const timeAgo = getTimeAgoFromNow(new Date(date))
+    setDateRun(timeAgo)
+  }
+
+  useEffect(() => {
+    if (date !== '') {
+      recalculateTimeAgo()
+      setInterval(recalculateTimeAgo, 60000)
```

Comment:
nice! - do we also need to clear the interval on "unmount"? https://stackoverflow.com/questions/55763957/how-to-clean-up-setinterval-in-useeffect-using-react-hooks

Reply:
Ohh!! of course, that's very important. Done [3068412](https://github.com/mergestat/fuse/pull/89/commits/3068412459b5ee7151a5b88ad0dc5e880f43f9b0)